airflow/utils/db.py

```diff
@@ -245,6 +245,17 @@ def initdb():
                     ]
                 }
             '''))
+    merge_conn(
```

Comment:
Why is this added? It won't work for other users than yourself? If it is for testing purposes, why don't you just add it in the setup part of the tests?